test/unit/asset-fs-extra/output.js

```diff
@@ -0,0 +1,57 @@
+[
+  "node_modules/fs-extra/lib/copy-sync/copy-file-sync.js",
```

Comment:
I was a bit surprised to see all these files in `actual.js`, but then I found the same pattern in `test/unit/pixelmatch/output.js` and `test/unit/webpack-wrapper-strs-namespaces-large/output.js`.

Reply:
Yes, this expected since `fs-extra` needs to be traced too since its not a native dependency